[2025-06-16 22:58:34,110] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: load_clean_data_ecommerce.clean_data 2025-06-16T22:57:38.118091+00:00 [queued]>
[2025-06-16 22:58:34,125] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: load_clean_data_ecommerce.clean_data 2025-06-16T22:57:38.118091+00:00 [queued]>
[2025-06-16 22:58:34,125] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2025-06-16 22:58:34,126] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2025-06-16 22:58:34,126] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2025-06-16 22:58:34,150] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): clean_data> on 2025-06-16T22:57:38.118091+00:00
[2025-06-16 22:58:34,158] {standard_task_runner.py:52} INFO - Started process 237 to run task
[2025-06-16 22:58:34,166] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'load_clean_data_ecommerce', 'clean_data', '2025-06-16T22:57:38.118091+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Load_data_dag.py', '--cfg-path', '/tmp/tmprol9lmnj', '--error-file', '/tmp/tmpd5iflb5l']
[2025-06-16 22:58:34,166] {standard_task_runner.py:77} INFO - Job 4: Subtask clean_data
[2025-06-16 22:58:34,243] {logging_mixin.py:104} INFO - Running <TaskInstance: load_clean_data_ecommerce.clean_data 2025-06-16T22:57:38.118091+00:00 [running]> on host dd40c2b2cc41
[2025-06-16 22:58:34,410] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=load_clean_data_ecommerce
AIRFLOW_CTX_TASK_ID=clean_data
AIRFLOW_CTX_EXECUTION_DATE=2025-06-16T22:57:38.118091+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-06-16T22:57:38.118091+00:00
[2025-06-16 22:58:37,841] {python.py:151} INFO - Done. Returned value was: None
[2025-06-16 22:58:37,856] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=load_clean_data_ecommerce, task_id=clean_data, execution_date=20250616T225738, start_date=20250616T225834, end_date=20250616T225837
[2025-06-16 22:58:37,903] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-16 22:58:37,939] {local_task_job.py:151} INFO - Task exited with return code 0
